src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -50,11 +50,10 @@ public partial class PackageManagerControl : UserControl, IVsWindowSearch
 
         // Signifies where an action is being executed. Should be updated in a coordinated fashion with IsEnabled
         private bool _isExecutingAction;
-
-        private PackageRestoreBar _restoreBar;
-
         private RestartRequestBar _restartBar;
 
+        public PackageRestoreBar RestoreBar { get; private set; }
```

Comment:
`RestoreBar` doesn't appear to be used outside this class. Why was it made public?

Reply:
It already was public. 
I just used auto property instead of a field and prop. 

It's used in the Apex tests.